portfolio/src/main/webapp/index.html

```diff
@@ -1,30 +1,46 @@
 <!DOCTYPE html>
 <html>
+
   <head>
     <meta charset="UTF-8">
     <title>My Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
-  <body>
-
-    <form action="/form-handler" method="POST">
-        <p>Enter some text here:</p>
-        <textarea name="text-input">hello world</textarea>
-        <br/>
-        <input type="submit" />
-      </form>
-      
+  
+  <body class="gradient-background">
     <div id="content">
       <h1>Erica Chen</h1>
       <p>This is my portfolio.</p>
       <img src="/images/pfp.PNG" 
-      width="200" 
-      height="400"/>
+      width="225" 
+      height="300"/>
       <p>Click <a href="/hello">here</a> to see content created by server code.</p>
       <p>Click here to get a random message:</p>
       <button onclick="addRandomGreeting()">Hello</button>
       <div id="greeting-container"></div>
+    
+      <form action="/form-handler" method="POST">
+        <p> Enter text: </p>
+        <textarea name="text-input"> hello world </textarea>
+        <br/>
+        <input type="submit" />
+      </form>
+
+      <form action='/sentiment' method="POST">
+        <p> Enter text to be analyzed: </p>
+        <textarea name="message"> good stuff </textarea>
+        <br/>
+        <button> Submit </button>
+      </form>
```

Comment:
This also does not work

```
HTTP ERROR 500 java.lang.AbstractMethodError: Receiver class com.google.api.gax.grpc.InstantiatingGrpcChannelProvider does not define or inherit an implementation of the resolved method 'abstract com.google.api.gax.rpc.TransportChannelProvider withExecutor(java.util.concurrent.Executor)' of interface com.google.api.gax.rpc.TransportChannelProvider.
```



URI: | /sentiment
-- | --
500
java.lang.AbstractMethodError: Receiver class com.google.api.gax.grpc.InstantiatingGrpcChannelProvider does not define or inherit an implementation of the resolved method 'abstract com.google.api.gax.rpc.TransportChannelProvider withExecutor(java.util.concurrent.Executor)' of interface com.google.api.gax.rpc.TransportChannelProvider.
com.google.sps.servlets.SentimentAnalysis
java.lang.AbstractMethodError: Receiver class com.google.api.gax.grpc.InstantiatingGrpcChannelProvider does not define or inherit an implementation of the resolved method 'abstract com.google.api.gax.rpc.TransportChannelProvider withExecutor(java.util.concurrent.Executor)' of interface com.google.api.gax.rpc.TransportChannelProvider.

<br class="Apple-interchange-newline">